### ISSUE TITLE
Bug : picture edited on iOS localy not seen edited in the picker !

### ISSUE THREAD

**Issue by VuillaumeGautier** [weta-vn/advance_image_picker#25]
Hello again !

We found a bug while using the app : 
on iOS, when you edit a picture in the iOS image editor, the picture is seen croped etc. in the iOS gallery, but not in the image_picker gallery.

Tell me f you need more info for the troubleshooting.

Thanks again for the lib and your time !

**Comment by rydmike** [comment]
Perhaps a bit more explanation of what you are seeing and what expect too se would help.

The advance_image_picker will not update images you edit in the image picker to the device media photo/gallery. It creates temp edited cropped images from photos taken or selected from gallery. The edits are applied to temp copies that are returned. If you would then like to save these edited/cropped/stickered  image back to the gallery, your app would have to do so or whatever else you want to do with the returned images.

If on the other hand **(I think you mean this case)** you are using an iOS device and its native image editor, to edit images in the device gallery, and those edits are not seen in the advance_image_picker gallery's media library, then there might indeed be a caching or similar issue.

**Comment by VuillaumeGautier** [comment]
@rydmike You're right with the second case. The first case is not expected nor an issue whatsoever for us, but the second one is a bit annoying (though it just have been spotted in the team).

**Comment by rydmike** [comment]
Thanks for confirming the issue details. I have seen references to a similar issue for other usage than this particular picker. I think it might be related to that and that it is indeed a caching issue on iOS with the media library usage. I will see what more info I can dig up on it, then we can look into what we can do here to address it.

**Comment by weta-vn** [comment]
@VuillaumeGautier 
Thank you.
We confirmed this issue, and try to analyze root cause but no easy.
The advance image picker use photo_manager package to get assets from ios library, but that seem can not get edited photo path.
We'll investigate more about this.

**Comment by rydmike** [comment]
We might need to open an issue with the photo_manager package, the solution to this issue is perhaps best addressed there, although I did not have time to look at the dentist yet. That was just my initial reaction when I looked at it briefly.

**Comment by AlexV525** [comment]
We have applied multiple fixes around edited resources, especially on iOS, including:
- https://github.com/fluttercandies/flutter_photo_manager/pull/662
- https://github.com/fluttercandies/flutter_photo_manager/pull/663
- https://github.com/fluttercandies/flutter_photo_manager/pull/683

So I'm assuming the issue has been fixed, and they will be published in the 2.0.0 version of `photo_manager`.

Feel free to leave feedback in our repo tho.
